src/Tribe/CSV_Importer/Rows.php

```diff
@@ -32,7 +32,7 @@ public static function instance() {
 	 * @return array
 	 */
 	public function filter_import_options_rows( array $import_options ) {
-		$import_options['rsvp'] = esc_html_x( 'RSVPs', 'import type', 'event-tickets' );
+		$import_options['rsvp'] = tribe_get_rsvp_label_plural( 'import_type' );
```

Comment:
Can you verify this escaping is done in the usage of the import options further down the line?